src/PermanentFilterProvider.tsx

```diff
@@ -40,19 +41,22 @@ export const PermanentFilterProvider = ({ children }: { children?: React.ReactNo
     mowasRegionalKeysReducer,
     []
   );
-  const [filterState, filterDispatch] = useReducer(overlayFilterReducer, []);
+  const [resourceFilterState, resourceFilterDispatch] = useReducer(resourceFilterReducer, []);
 
   const loadFilters = useCallback(async () => {
     const dataProviderIds = ((await readFromStore(DATA_PROVIDER_FILTER_KEY)) ?? []) as string[];
     const mowasRegionalKeys = ((await readFromStore(MOWAS_REGIONAL_KEYS)) ?? []) as string[];
-    const overlayFilter = ((await storageHelper.filter()) ?? []) as string[];
+    const resourceFilter = ((await readFromStore(RESOURCE_FILTER_KEY)) ?? []) as string[];
```

Comment:
we save multiple resource filters, right? so i suggest naming all the consts in plural